src/pages/Register.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <ion-page>
+    <Toolbar title-text = "Register"/>
+
+    <ion-content :fullscreen="true">
+      <div id="login">
```

Comment:
please use meaningful names. Your div here is not the "login", rather a container of the form.